plugins/snippets/luasnip/default.nix

```diff
@@ -50,22 +50,33 @@ let
   };
 in
 {
+  imports =
+    let
+      basePluginPath = [
+        "plugins"
+        "luasnip"
+      ];
+    in
+    [
+      (lib.mkRenamedOptionSubmodule (basePluginPath ++ [ "extraConfig" ]) (
```

Comment:
This should probably have a TODO added/remove comment

Reply:
Indeed, done